docs/wmco-prerequisites.md

```diff
@@ -0,0 +1,51 @@
+# Cluster and OS Pre-Requisites
+Below are the Cluster and OS prerequisites for WMCO. Please also see the vSphere-specific section that 
+may be relevant.
+
+## Supported Cloud Providers based on OKD/OCP Version and WMCO version
+| Cloud Provider           | Supported OKD/OCP Version   | Supported WMCO version   |
+| -----------              | -----------                 | -----------              |
+| Amazon Web Services (AWS)| 4.6+                        | WMCO 1.0+                |
+| Azure                    | 4.6+                        | WMCO 1.0+                |
+| VMware vSphere           | 4.7+                        | WMCO 2.0+                |
+
+## Supported Windows Server versions
+The following table outlines the supported
+[Windows Server version](https://docs.microsoft.com/en-us/windows/release-health/release-information) based on the 
+applicable cloud provider.
+
+Note: Any unlisted Windows Server version are NOT supported, and will cause errors. To prevent 
+these errors, only use the appropriate version according to the cloud provider in use. 
+
+| Cloud Provider      | Supported Windows Server version                                        |
+| -----------         | -----------                                                             |
+| AWS                 | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809  |
+| Azure               | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809  |
+| VMware vSphere      | Windows Server Semi-Annual Channel (SAC): Windows Server 2004           |
+
+## Supported Networking
+[OVNKubernetes hybrid networking](setup-hybrid-OVNKubernetes-cluster.md) is the only supported networking configuration.
+The following tables outline the type of networking configuration and Windows Server versions to be used based on your 
+cloud provider. This must be completed during the installation of the cluster.
```

Comment:
s/This/The network configuration/ to clear. We don't want the user to think there is an option to bring up a Windows node during cluster install.